docs/sources/gard.md

```diff
@@ -1,17 +1,90 @@
 # MONDO - GARD Alignment
 
-**Source name:** GARD
+**Source name:** Genetic and Rare Diseases
 
-**Source description:** The GARD mapping is mainly curated through the GARD-ORDO and GARD-OMIM mappings. From the GARD developers, we have confirmed that, For a given GARD ID,  GARD guarantees no other concept will be assigned in the future.   The concept will not be assigned to a different GARD ID, but this GARD ID may be retired  when a similar but more accurate concept is created under a new GARD ID to replace this GARD ID.
+**Source description:**  Genetic and Rare Diseases (GARD) Information Center is a public health resource that aims to support people living with a rare disease and their families with free access to reliable, easy to understand information. 
 
+**Homepage:** https://rarediseases.info.nih.gov/
 
-**Homepage:** None
+## Comments about this source:
+### [README](https://docs.google.com/document/d/1sM0B0ICdOWDwZWt0M_Y3t5QYwqL-XwOsDl3TS14XRa4/edit#): GARD::Mondo mappings analysis
+#### About the [spreadsheet](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1607652739)
+1. `gard.sssom+`: This is a SSSOM of GARD mappings. For how predicate_id’s were chosen, see “Assumptions” section below.
+Additional columns added: mondo_id, mondo_label, and gard_mondo_mapping_type (which has 3 values: `direct_existing` (
+taken from [mondo_hasdbxref_gard.sssom.tsv](https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo_hasdbxref_gard.sssom.tsv))
+, `new_proxy`, and `unmapped_proxy_or_direct` (which is same as the `gard_unmapped_terms` sheet). 
+2. `gard_unmapped_terms`: A flat list of GARD IDs that do not have existing mappings in Mondo, and no new proxy mappings found.
+3. `GARD source data`: Copy of source CSV provided by GARD.
+4. `gard_terms_mapping_status`: This output was mainly for us to provide to the GARD folks. This allows us to see at a 
+glance the GARD IDs that exist in the GARD data source as well as in our current Mondo mappings.
+5. `obsoleted_gard_terms_in_mondo`: This output was initially for us to provide to the GARD folks.This is a flat list of
+ GARD IDs that have xrefs in Mondo but are no longer in GARD.
 
-**Comments about this source:** None
+#### Assumptions
+**Parent/Child relationships**  
+The [GARD source data](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=885755208)
+has a ParentGardID column is formatted just like a Python list, comma delimited, with opening and closing brackets 
+(`[` and `]`). All IDs here are considered instances of `rdfs:subClassOf` for the given record.
 
+**Mappings**  
+Joe and Nico interpreted the [GARD source data](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=885755208) 
```

Comment:
Better avoid links to google docs/sheets and refer to checked in files in the gard repo